django/db/backends/base/base.py

```diff
@@ -205,13 +206,15 @@ def check_database_version_supported(self):
 
     # ##### Backend-specific methods for creating connections and cursors #####
 
+    @sensitive_variables()
```

Comment:
This method doesn't get connection parameters and has to be implemented in the database backends. What's the point in adding a decorator here? Almost the same with `get_new_connection()` it will be reimplemented by database backends so there is no point in adding a decorator to the base implementation. It seems that you didn't try your solution :confused: